script.js

```diff
@@ -0,0 +1,21 @@
+function palindrom(){
+    let word=document.getElementById("word").value;
+    word_to_check=word.toLowerCase();
+    let length_word=word_to_check.length;
+    let sprawdz=true;
+    if (length_word===0)                
+        alert("Nic nie wpisałeś!");   
+        else {
+    for (var i=0; i<length_word; i++ ) 
```

Comment:
Do we need to check all of the world?

Eg in "kayak" you don't have to check 1 and 5 letter and then 5 and 1 as well.

Try to optimize it ;)